### ISSUE TITLE
add RPM spec template

### ISSUE THREAD

**Issue by andy5995** [theimpossibleastronaut/rmw#268]
Support for an `rpm dist` target can be added with the autoconf macro 

`AX_DIST_RPM([SPEC])`

However,

> You will need to create a spec template with everything except the files and the Changelog. Use @PACKAGE@ and @VERSION@ to refer to the package name and version respectively. The files and ChangeLog will be filled in automatically. For instance: 

See https://www.gnu.org/software/autoconf-archive/ax_dist_rpm.html#ax_dist_rpm for more details

**Comment by blackfile** [comment]
@andy5995  I can help with this issue

**Comment by andy5995** [comment]
Hi @blackfile 

No spec file exists yet. It can go in the `packaging` directory.

**Comment by andy5995** [comment]
The [spec template was added](https://github.com/theimpossibleastronaut/rmw/commit/2a3b72571a38459b49f4dae6fb9dfbe02324bed8) by @blackfile  but we're not going to add a `dist-rpm` target.. autotools requires too many extra files.

rpmbuild should be able to use it, but it fails when "-dev" is appended to the version name. Also, not all files are being included when I tried temporarily changing the version to just 0.7.05.

So needs some work for a finished product.
